src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1860,7 +1864,274 @@ void updateZoneSizingEndDay(DataSizing::ZoneSizingData &zsCalcSizing,
+    for (int ts = 1; ts <= state.dataZoneEquipmentManager->NumOfTimeStepInDay; ++ts) {
+        zoneCFS.HeatFlowSeq(ts) = 0.0;
+        zoneCFS.HeatLoadSeq(ts) = 0.0;
+        zoneCFS.HeatZoneTempSeq(ts) = 0.0;
+        zoneCFS.HeatOutTempSeq(ts) = 0.0;
+        zoneCFS.HeatZoneRetTempSeq(ts) = 0.0;
+        zoneCFS.HeatZoneHumRatSeq(ts) = 0.0;
+        zoneCFS.HeatOutHumRatSeq(ts) = 0.0;
+        zoneCFS.HeatLoadNoDOASSeq(ts) = 0.0;
+        zoneCFS.CoolFlowSeq(ts) = 0.0;
+        zoneCFS.CoolLoadSeq(ts) = 0.0;
+        zoneCFS.CoolZoneTempSeq(ts) = 0.0;
+        zoneCFS.CoolOutTempSeq(ts) = 0.0;
+        zoneCFS.CoolZoneRetTempSeq(ts) = 0.0;
+        zoneCFS.CoolZoneHumRatSeq(ts) = 0.0;
+        zoneCFS.CoolOutHumRatSeq(ts) = 0.0;
+        zoneCFS.CoolLoadNoDOASSeq(ts) = 0.0;
+    }
+    if (zoneCFS.zoneLatentSizing) {
+        // Zero out latent simple sums
+        zoneCFS.DesLatentHeatVolFlow = 0.0;
+        zoneCFS.DesLatentHeatMassFlow = 0.0;
+        zoneCFS.DesLatentHeatLoad = 0.0;
+        zoneCFS.DesLatentHeatLoadNoDOAS = 0.0;
+        zoneCFS.DesLatentCoolVolFlow = 0.0;
+        zoneCFS.DesLatentCoolMassFlow = 0.0;
+        zoneCFS.DesLatentCoolLoad = 0.0;
+        zoneCFS.DesLatentCoolLoadNoDOAS = 0.0;
+
+        // Zero out latent weighted averages
+        zoneCFS.ZoneTempAtLatentHeatPeak = 0.0;
+        zoneCFS.ZoneHumRatAtLatentHeatPeak = 0.0;
+        zoneCFS.ZoneRetTempAtLatentHeatPeak = 0.0;
+        zoneCFS.DesLatentHeatCoilInTemp = 0.0;
+        zoneCFS.DesLatentHeatCoilInHumRat = 0.0;
+        zoneCFS.ZoneTempAtLatentCoolPeak = 0.0;
+        zoneCFS.ZoneHumRatAtLatentCoolPeak = 0.0;
+        zoneCFS.ZoneRetTempAtLatentCoolPeak = 0.0;
+        zoneCFS.DesLatentCoolCoilInTemp = 0.0;
+        zoneCFS.DesLatentCoolCoilInHumRat = 0.0;
+
+        // Zero out latent time-series sums
+        for (int ts = 1; ts <= state.dataZoneEquipmentManager->NumOfTimeStepInDay; ++ts) {
+            zoneCFS.LatentHeatLoadSeq(ts) = 0.0;
+            zoneCFS.LatentHeatFlowSeq(ts) = 0.0;
+            zoneCFS.HeatLatentLoadNoDOASSeq(ts) = 0.0;
+            zoneCFS.LatentCoolLoadSeq(ts) = 0.0;
+            zoneCFS.LatentCoolFlowSeq(ts) = 0.0;
+            zoneCFS.CoolLatentLoadNoDOASSeq(ts) = 0.0;
+        }
+    }
+
+    int numSpaces = 0; // Track this for averages later
+    for (int spaceNum : state.dataHeatBal->Zone(zoneNum).spaceIndexes) {
+        auto &spaceCFS = state.dataSize->CalcFinalSpaceSizing(spaceNum);
+        ++numSpaces;
+
+        // Simple sums
+        zoneCFS.DesHeatVolFlow += spaceCFS.DesHeatVolFlow;
+        zoneCFS.DesHeatLoad += spaceCFS.DesHeatLoad;
+        zoneCFS.DesHeatMassFlow += spaceCFS.DesHeatMassFlow;
+        zoneCFS.DesHeatLoadNoDOAS += spaceCFS.DesHeatLoadNoDOAS;
+        zoneCFS.DesCoolVolFlow += spaceCFS.DesCoolVolFlow;
+        zoneCFS.DesCoolLoad += spaceCFS.DesCoolLoad;
+        zoneCFS.DesCoolMassFlow += spaceCFS.DesCoolMassFlow;
+        zoneCFS.DesCoolLoadNoDOAS += spaceCFS.DesCoolLoadNoDOAS;
```

Comment:
For the non-coincident case, why is the zone data getting overwritten by the space data? I understand wanting to average the space data in that case but why overwrite to zone data? If the user selects non-coincident sizing the zone sizing information will be lost. If I had seen this I would be more comfortable:

     zoneCFS.spaceDesHeatVolFlow += spaceCFS.DesHeatVolFlow;    or
     zoneSpaceCFS.DesHeatVolFlow += spaceCFS.DesHeatVolFlow;

So for the non-coincident case the data written to eplusout.zsz won't be what it was before. I thought there was going to be 2 independent reports for zone and space sizing and those reports could be compared to see how well space sizing agreed with zone sizing.

Reply:
This might actually be OK because you could compare eplusout.zsz for the coincident and non-coincident cases and if there were no diffs, or very small diffs, then keeping both separate would not be an issue. If there are big diffs then?

Reply:
I see 0 diffs between coincident and non-coincident results of eplusout.zsz. Carry on (I think I got these diffs right). For non-coincident I added this to the end of Sizing:Zone:

    autosize,                !- Dedicated Outdoor Air High Setpoint Temperature for Design {C}
    ,
    ,
    ,
    ,
    ,
    ,
    ,
    ,
    ,
    NonCoincident;            !- Type of Space Sum to Use


![image](https://github.com/user-attachments/assets/d240039d-413b-406b-b3a9-9b2957ea0ed5)

Reply:
The current logic is this:
1. Let existing code calculate the space sizing and coincident zone sizing (i.e. lump spaces together for the zone sizing).
2. If non-coincident sizing is selected for a given zone, then override the zone sizing results based on the sums/averages of spaces in that zone.
3. However, if the zone only has one space, then skip the override and keep the original results. That's why you didn't seen any diffs in 5ZoneAirCooledWithSpaceHeatBalance (it's only got one space per zone).
4. Here's the zsz files for 5ZoneAirCooledWithSpacesHVAC. There are differences for zones 3 and 5. Looks like the peaks are not correct, however - the difference in peaks shouldn't be greater than the hourly diffs.
[5ZoneAirCooledWithSpacesHVAC-NonCoincident.zip](https://github.com/user-attachments/files/16350439/5ZoneAirCooledWithSpacesHVAC-NonCoincident.zip)

Reply:
This logic seems sound. I don't see the problem with non-coincident vs zone sizing as you mentioned. I was worried that overwriting data may cover up an issue but the user can always compare coincident vs non-coincident to verify results.

![image](https://github.com/user-attachments/assets/15015120-11a8-4134-aa3e-d1cec06022f0)

Reply:
I still need to figure out what to do with peak date/time. I guess I can scan the non-coincident zone seqs?

Reply:
But those could have come from different design days for each space, hmm.